src/test/java/data/DatabaseTest.java

```diff
@@ -184,7 +184,7 @@ public void testGetByBuilding()
 		//Verify that the building was actually inserted
 		final String uuid = database.getBuildingUUID("Starfleet Headquarters");
 		assertEquals(36, uuid.length());
-		assertEquals(1, database.getBuildings().size());
+		assertEquals(3, database.getBuildings().size());
```

Comment:
This test case shouldn't pass, the DatabaseTest class uses its own separate test database that is independent of our reset data. If this doesn't pass, it needs to be reverted. If it DOES pass, there's a very unpleasant bug lurking somewhere.